src/shell/menus.cpp

```diff
@@ -621,9 +621,11 @@ OrderDeliverMenu::On_Menu_Init()
   attroff(A_BOLD);
 
   for (size_t i = 0; i < items_.size(); ++i) {
-    // We must substract 1 because items_[i][1] contains the ID of the order,
-    // but oinfo_->orders is an array starting at 0
-    const llsf_msgs::Order &o = oinfo_->orders(std::get<1>(items_[i]) - 1);
+    const llsf_msgs::Order &o = *std::find_if(oinfo_->orders().begin(),
```

Comment:
This direct dereference could fail if the order is not found.

Reply:
Fixed, also in other other place, where the same issue may occur